include/uv-win.h

```diff
@@ -45,8 +45,8 @@ typedef struct pollfd {
 
 #ifndef _malloca
 # if defined(_DEBUG)
-#  define _malloca(size) malloc(size)
-#  define _freea(ptr) free(ptr)
+#  define _malloca(size) uv__malloc(size)
+#  define _freea(ptr) uv__free(ptr)
```

Comment:
Does this actually work?  uv-win.h is a semi-public header, it gets included by the downstream project, whereas `uv__malloc()` and `uv__free()` are internal.

For that matter, I wonder why these defines are in uv-win.h in the first place.  Maybe this block should just be removed?  @piscisaureus?